core/modules/system/system.api.php

```diff
@@ -3039,15 +3039,59 @@ function hook_disable() {
 }
 
 /**
+ * Provide paths to classes and interfaces that can be autoloaded by PHP.
+ *
+ * Most classes and interfaces in Backdrop are autoloaded, meaning they do not
+ * need to be manually included with PHP's include_once() or require_once().
+ * Creating a new instance of a class will automatically include the needed
+ * file that contains that class.
```

Comment:
Can we shorten this up a bit?  Maybe something like this:
`Most classes and interfaces in Backdrop should be autoloaded. This will prevent
the need to manually include the file that contains that class with PHP's
include_once() or require_once().`